### DISCUSSION TITLE
Doesn't work on my Hp Probook 445 G7

### DISCUSSION THREAD

**Discussion by kxng1080xx** [DavidS95/Smokeless_UMAF#59] [Q&A]
Tried it but only network devices come up is there a fix or something else I can try

**Comment by angelokimhui** [comment]
Considering that the author seems to be inactive and does not interact github in discussions. I guess we're all just sitting duck here. Lol. I appreciate the effort but I wish we have some guide and progress here. Unfortunately, HP seems to be using something to block the loading of the form. I have different device though. HP Elitebook 845 G7.

**Reply by Sayor0630** [reply]
> Considering that the author seems to be inactive and does not interact github in discussions. I guess we're all just sitting duck here. Lol. I appreciate the effort but I wish we have some guide and progress here. Unfortunately, HP seems to be using something to block the loading of the form. I have different device though. HP Elitebook 845 G7.

I also have HP EliteBook 845 G7. but does not show anything in devices

**Reply by Vlad55432** [reply]
HP ProBook 445 G8 user here, only shows these options
(I'll act that it's not one year old stale discussion)
![PXL_20251003_005231_lmc_8 4](https://github.com/user-attachments/assets/818824d8-35d5-491d-ad69-a3aaba982080)
![PXL_20251003_011442_lmc_8 4](https://github.com/user-attachments/assets/9a81655a-2c6f-4511-879c-db1b742fd64a)
These shows up on launch
<img width="1080" height="2263" alt="Screenshot_20251003-012115_Photos" src="https://github.com/user-attachments/assets/d91b4209-5b30-4c3f-8849-612a12eb0b2a" />

**Reply by ajshovon** [reply]
> HP Elitebook 845 G7

any luck with your laptop?

**Comment by Cr1pter** [comment]
Habe das gleiche Problem auf meinem HP Probook 435 G8
